### DISCUSSION TITLE
William Wythers' Overhauled Overworld & Expanded Ecosphere

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#903] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/william-wythers-overhauled-overworld
+ Overhaul of vanilla biomes

https://www.curseforge.com/minecraft/mc-mods/expanded-ecosphere
+ Tens of biomes
+ Hundreds of trees

**Comment by Koteczek0088** [comment]
i think it would be nice to have this compat mod
